src/ui/helper/index.ts

```diff
@@ -365,3 +365,15 @@ export function findElByClassName(e: Event, className: string) {
     target : findParentBySelector(target, '.' + className)
 }
 
+export function externalLink(text: string, url: string): Mithril.Children {
+  return h('span', { class: 'link' }, [
+    h('a', { href: '#', onClick: 'window.open("' + url + '")' }, text),
+    h('span', { class: 'fa fa-external-link' })
```

Comment:
don't add this, instead use the `external_link` class on the `a` element.
see: https://github.com/veloce/lichobile/blob/master/src/utils/html.ts#L31